core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -625,6 +621,14 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
     <td>{errorSummary}{details}</td>
   }
 
+  private def getGettingResultTime(info: TaskInfo): Long = {
+    if (info.gettingResultTime > 0 && info.finishTime > 0) {
```

Comment:
PS I noticed when using `TaskInfo` recently that it has `gettingResult` and `finished` property that does this same check. I used them instead of also comparing the times to 0.